### ISSUE TITLE
Pending Badge doesn't display properly

### ISSUE THREAD

**Issue by elwayman02** [mixonic/ember-community-versions#14]
See screenshot below:

![jordan_hawker](https://cloud.githubusercontent.com/assets/1325482/9826377/91472e3a-588e-11e5-9e9a-82f7675dade5.png)

My badge code:

```
<iframe width="178" height="24" style="border:0px" src="https://mixonic.github.io/ember-community-versions/2015/09/07/upgrading-ember-the-way-it-was-intended-part-i.html">
</iframe>
```

FWIW, I was able to embed the sample "real" badge and have it show up fine. It was only the badge for non-existent jekyll posts that didn't work.

**Comment by raytiley** [comment]
yeah, seeing the same thing after installing non-existant badges.

**Comment by raytiley** [comment]
Looks like you need to use a custom domain. https://help.github.com/articles/custom-404-pages/

**Comment by mixonic** [comment]
@raytiley I think that is only for "Project Pages", this is a personal page.

**Comment by mixonic** [comment]
Hm, but yeah I don't see it either. Drat, perhaps we do need a custom domain then.

**Comment by raytiley** [comment]
@mixonic this is a project page. Its a page for "ember-community-versions" repo: https://help.github.com/articles/user-organization-and-project-pages/#project-pages

**Comment by elwayman02** [comment]
@mixonic I'd be happy to provide a sub-domain off my `jhawk.co` domain just to get this off the ground until we decide on a more permanent solution.  Maybe `version.jhawk.co`?
